src/librustc_mir/interpret/validity.rs

```diff
@@ -835,9 +835,12 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         // Run it.
```

Comment:
While we are adjusting the forwarding here, could you also adjust the comment `// Other errors shouldn't be possible` above (line 795) to match what we do elsewhere when propagating?
```rust
// Propagate other errors upwards (that will also check for unexpected errors).
```